go/cmd/dolt/commands/push.go

```diff
@@ -100,7 +102,10 @@ func (cmd PushCmd) Exec(ctx context.Context, commandStr string, args []string, d
 			}
 			verr = errhand.BuildDError("fatal: The current branch " + currentBranch.GetPath() + " has no upstream branch.\n" +
 				"To push the current branch and set the remote as upstream, use\n" +
-				"\tdolt push --set-upstream " + remoteName + " " + currentBranch.GetPath()).Build()
+				"\tdolt push --set-upstream " + remoteName + " " + currentBranch.GetPath() + "\n" +
+				"To have this happen automatically for branches without a tracking\n" +
+				"upstream, see 'push.autoSetupRemote' in 'dolt config --help'.").Build()
```

Comment:
Great error message! This will be super helpful for people to find this new config option.